admin/includes/functions/general.php

```diff
@@ -758,3 +758,31 @@ function zen_get_orders_status_pulldown_array()
 
     return $orders_status_array;
 }
+
+/***
+ * Do the misconfiguration check which Admin > Catalog > Downloads Manager 
+ * does to verify that downloads don't have invalid shipping settings.
+ */
+function zen_check_downloads_shipping() {
```

Comment:
Rename `zen_check_downloads_shipping()` to `zen_check_for_misconfigured_downloads()`

Reply:
And move to `functions_attributes.php`